### PR TITLE
Redesign 0.8.0

### PR DESCRIPTION
**Pull request by IliaSolovev**

### CODE REVIEW

File: src/components/breadcrumbs/breadcrumbs.tsx [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+  const { adminPath } = dividePathname(pathname);
+  const { buildVersion, scopeId } = useTestToCodeRouteParams();
+  const prepareLink = replaceLinkValues({ buildVersion, scopeId });
+  const prepareContent = getContent({ buildVersion, scopeId });
+
+  return (
+    <BreadcrumbsContainer>
+      {crumbs.map((crumb) => {
+        const rawLink = `${adminPath}${prepareLink(crumb.path)}`;
+        const link = prepareLink(rawLink);
```
Comment (RomanAchilov) [inline src/components/breadcrumbs/breadcrumbs.tsx]: link = prepareLink(`${adminPath}${prepareLink(crumb.path)}`) mb?

File: src/components/breadcrumbs/breadcrumbs.tsx [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+`;
+
+const CrumbLink = styled.div`
+  ${tw`inline-block max-w-200px
+      text-ellipsis align-middle
+      text-blue-default text-12
+      font-bold cursor-pointer no-underline
+  `};
+
+  :last-child {
```
Comment (RomanAchilov) [inline src/components/breadcrumbs/breadcrumbs.tsx:101]: https://tailwindcss.com/docs/hover-focus-and-other-states

File: src/components/breadcrumbs/breadcrumbs.tsx [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+    ${tw`text-monochrome-default pointer-events-none`};
+  }
+
+  :not(:first-child):before {
```
Comment (RomanAchilov) [inline src/components/breadcrumbs/breadcrumbs.tsx:105]: https://github.com/ben-rogerson/twin.macro

File: src/components/breadcrumbs/breadcrumbs.tsx [hunk 4]
```diff
@@ -0,0 +1,137 @@
...
+    ${tw`hover:text-blue-medium-tint active:text-blue-shade`};
+  }
+`;
+
+const dividePathname = (pathname: string) => {
+  const [adminPath = "", test2codePath = ""] = pathname.split(`/${PLUGIN_ID}/`);
+  return { adminPath: `${adminPath}/${PLUGIN_ID}`, test2codePath };
+};
+
+const replaceLinkValues = ({ buildVersion, scopeId }: {buildVersion: string, scopeId: string}) =>
```
Comment (RomanAchilov) [inline src/components/breadcrumbs/breadcrumbs.tsx:115]: Gl hf

File: src/pages/agent/build-overview/build-overview.tsx [hunk 5]
```diff
@@ -39,8 +39,9 @@ export const BuildOverview = () => {
     <>
       <CoveragePluginHeader />
       <div tw="flex mb-4 border-b border-monochrome-medium-tint">
+        {/* !activeTab expressions means that t is default active tab */}
         <Tab
-          active={activeTab === "methods"}
+          active={!activeTab || activeTab === "methods"}
```
Comment (RomanAchilov) [inline src/pages/agent/build-overview/build-overview.tsx]: Active not active tab 🙂

**Review by RomanAchilov — CHANGES_REQUESTED** [review]

**Comment by IliaSolovev** [comment]
rebase

**Pull request merged**